test/test_helper.rb

```diff
@@ -148,6 +148,50 @@ def init_annotation(org, question, **props)
       nil
     end
   end
+  

```

Comment:
good to see this helpers however I'm concern about the length this file is taking. We should split into multiple files so that it makes more maintainable the testing